src/mam4xx/utils.hpp

```diff
@@ -209,7 +217,9 @@ void extract_stateq_from_prognostics(const mam4::Prognostics &progs,
   for (int m = 0; m < AeroConfig::num_modes(); ++m) {
     // First add the aerosol species mmr
     for (int a = 0; a < mam4::num_species_mode(m); ++a) {
-      q[s_idx] = progs.q_aero_i[m][a](klev);
+      const int h_ind = get_haero_prognostics_index(
```

Comment:
Why do you name this function `get_haero_prognostics_index `? is the prognostics struct defined in `mam4xx`. should we call this function `get_prognostics_index `?

Reply:
I used `haero` in the name as `prognostics` is defined in `haero` (see [here](https://github.com/eagles-project/haero/blob/720bd29e8884eeba8dfcc9fc75476d5c6984ca85/haero/aero_process.hpp#L25))

I will change the name as adding `haero` doesn't add any new info the the function (we have just one `prognostics`)